scene/main/node.h

```diff
@@ -108,7 +108,8 @@ class Node : public Object {
 	enum NameCasing {
 		NAME_CASING_PASCAL_CASE,
 		NAME_CASING_CAMEL_CASE,
-		NAME_CASING_SNAKE_CASE
+		NAME_CASING_SNAKE_CASE,
+		NAME_CASING_KEBAB_CASE
```

Comment:
```suggestion
		NAME_CASING_KEBAB_CASE,
```

Reply:
Doh! I should have been paying more attention to the surrounding Enums. Thanks!